cypress/components/action-popover/action-popover.cy.tsx

```diff
@@ -316,8 +326,8 @@ context("Test for ActionPopover component", () => {
     );
 
     it.each([
-      ["Sub Menu 1", 0],
-      ["Sub Menu 2", 1],
+      [subMenuOption[0], 0],
+      [subMenuOption[1], 1],
     ])(
       "should close %s and ActionPopover after clicking on the submenu",
       (name, item) => {
```

Comment:
Same as previous comment.

Reply:
It's only used in the title of the test.